kubernetes/skipper-server/server-uri.sh

```diff
@@ -1,13 +1,10 @@
 if [ -z "$SKIPPER_SERVER_URI" ] ;
 then
   SKIPPER_IP=$(kubectl get svc --namespace $KUBERNETES_NAMESPACE | grep skipper | awk '{print $4}')
-  if [[ "$SKIPPER_IP" =~ [\d{1-3}:\d{1-3}:\d{1-3}:\d{1-3}] ]] ;
-  then
-       export SKIPPER_SERVER_URI="http://$SKIPPER_IP"
-       wait_for_200 ${SKIPPER_SERVER_URI}/api/about
-  else
-      echo "Unable to determine skipper server host $SKIPPER_IP Did deployment fail?"
-      exit 1
-  fi
+  while [[ "$SKIPPER_IP" == "<pending>" ]] ; do
```

Comment:
Why did you change this?

Reply:
We changed this together with @chrisjs because the old conditions were bogus and were letting the `SKIPPER_SERVER_URI` be set to `http:<pending>`. That in turn meant that the `wait_for_200 ${SKIPPER_SERVER_URI}/api/about` check fails.
The change above waits until the LoadBalancer for the Skipper is set and valid external IP is produced. 
Same change is applied to the SCDF script as well.

Reply:
side note; the external ip field may not always be an IP but rather a hostname on say AWS